src/net_processing.cpp

```diff
@@ -864,23 +864,18 @@ void PeerManager::ReattemptInitialBroadcast(CScheduler& scheduler) const
 }
 
 void PeerManager::FinalizeNode(NodeId nodeid, bool& fUpdateConnectionTime) {
-    fUpdateConnectionTime = false;
+    PeerRef peer = RemovePeer(nodeid);
```

Comment:
This undoes this assumption: https://github.com/bitcoin/bitcoin/pull/19607#discussion_r467079107 , which I'm still not convinced is a good idea.

Reply:
ok, removed that commit